adb-vagrantfile-tests/k8s-tests/run.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/python
+
+import json
+import urllib
+import subprocess
+import sys
+import os
+
+
+url_base = "http://admin.ci.centos.org:8080"
+api_key = os.environ['API_KEY']
+count = os.environ['MACHINE_COUNT']
+ver = "7"
+arch = "x86_64"
+req_url = "%s/Node/get?key=%s&ver=%s&arch=%s&count=%s" \
+    % (url_base, api_key, ver, arch, count)
+
+ansible_repo_url = os.environ['ANSIBLE_REPO_URL']
+
+test_cmd = "export ghprbActualCommit=%s && " % os.environ["ghprbActualCommit"]
+test_cmd += "export ghprbPullId=%s && " % os.environ["ghprbPullId"]
+test_cmd += os.environ['TEST_CMD']
+
+jsondata = urllib.urlopen(req_url).read()
```

Comment:
I'm okay to do that. However, the CentOS CI infra guys make sure that if environment is up, it's up as a whole and not in bits and pieces. That's been my experience so far. I can see the kind of situation we'd like to handle with `try..except`, but chances are it wouldn't come up. Anyway, point noted and implemented.